Python/other/left_nonRepeating_char.py

```diff
@@ -0,0 +1,44 @@
+"""Algorithm to find the first non-repeating character in a string
+
+
+Input the string from the user.
+Start traversing the string using two loops.
+Use the first loop to scan the characters of the string one by one.
+Use the second loop to find if the current character is occurring in the latter part if the string or not.
+If it is not occurring, print that character.
+Else, continue traversing.

```

Comment:
Add space and time complexity of the algorithm used by you.

Reply:
@atarax665 Complexities added.